kymatio/scattering1d/backend/numpy_backend.py

```diff
@@ -98,4 +98,10 @@ def ifft(cls, x):
 
         return cls._fft.ifft(x)
 
+    @classmethod
+    def transpose(cls, x):
```

Comment:
rename to `swap_time_frequency` ?

Reply:
i propose renaming because our implementation of `transpose` does a `complex_check`
it also does not exactly behave like `np.transpose`

Reply:
edited